test/connection/connectionSpec.ts

```diff
@@ -289,6 +289,7 @@ describe('connection', () => {
     await sendChallengeResponse()
     await receiveChallengeAccept()
     await receiveConnectionError()
+    await BBPromise.delay(10)
```

Comment:
you could use some variable to store the 10 value (`reconnectIntervalIncrement` for example) and use this in more places. This should make it easy to understand what '10'  means in this test.